Monika After Story/game/0utils.rpy

```diff
@@ -1,3 +1,275 @@
+python early:
+    import singleton
+    me = singleton.SingleInstance()
```

Comment:
Can this be in `0config.rpy`? That way it'd be defined as early as we can it make.